src/types.ts

```diff
@@ -53,6 +81,86 @@ export interface FrameStartType {
   sceneFrameCounter: number | null;
 }
 
+export interface GameInfoType {
+  gameBitfield1: number | null;
+  gameBitfield2: number | null;
+  gameBitfield3: number | null;
+  gameBitfield4: number | null;
+  bombRainEnabled: boolean | null;
+  itemSpawnBehavior: number | null;
+  selfDestructScoreValue: number | null;
+  startingTimerFrames: number | null;
+  itemSpawnBitfield1: number | null;
+  itemSpawnBitfield2: number | null;
+  itemSpawnBitfield3: number | null;
+  itemSpawnBitfield4: number | null;
+  itemSpawnBitfield5: number | null;
+  damageRatio: number | null;
+}
+
+export enum TimerType {
+  NONE = 0b00,
+  DECREASING = 0b10,
+  INCREASING = 0b11,
+}
+
+export enum ItemSpawnBehaviorType {
```

Comment:
```suggestion
export enum ItemSpawnType {
```
And can we shorten this one too?